packages/web-components/src/components/video-player/video-player.ts

```diff
@@ -15,6 +15,25 @@ import FocusMixin from 'carbon-web-components/es/globals/mixins/focus';
 const { prefix } = settings;
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
+/**
+ * @param options The options, with a video name and a formatted video duration.
+ * @returns The formatted video caption.
+ */
+export function formatCaption({ duration, name }: { duration?: string; name?: string }) {
+  return !name || typeof duration === 'undefined' ? name || duration || '' : `${name} (${duration})`;
+}
+
+/**
+ * @param options The options, with a video duration.
+ * @returns The formatted video duration.
+ */
+export function formatDuration({ duration }: { duration?: number }) {
+  const minutes = Math.floor((duration ?? 0) / 60);
+  const seconds = Math.floor((duration ?? 0) % 60);
+  const fillSeconds = Array.from({ length: 2 - String(seconds).length + 1 }).join('0');
+  return typeof duration === 'undefined' ? undefined : `${minutes}:${fillSeconds}${seconds}`;
+}
```

Comment:
I'm wondering if we should move this to the service, so this logic is shared by any framework?

Reply:
Good call @jeffchew - Probably utilities given the logic is not associated with any service (though the use case is of video, it essentially is a duration g11n formatter)?

Reply:
Utilities makes sense to me!

Reply:
Sounds good - Updated.

Reply:
Thank you for making the change! We should eventually have React use the same logic so it can be centralized, but that's low priority at the moment.